README.md

```diff
@@ -129,6 +129,16 @@ Example:
     $target.hasClass('disabled')
 ```
 
+## Dev Console
+
+To get the current context in the dev console inspect and element then type:
```

Comment:
Typo on the "and," and perhaps add a comma? "To access the current context in the dev console, inspect an element, then run:"